src/client/network.rs

```diff
@@ -371,6 +387,7 @@ impl KafkaConnection {
                id: id,
                host: host.to_owned(),
                stream: stream,
+               closed: false,
```

Comment:
A `TcpStream` is not guaranteed to be open, so I'm not sure it's correct to assume it is here.